### ISSUE TITLE
Claim: distilbert-base-uncased-finetuned-sst-2-english HuggingFace Model

### ISSUE THREAD

**Issue by Rutam21** [DagsHub/open-source-ml-models#12]
This issue aims to add the following ML Model to DagsHub.

# distilbert-base-uncased-finetuned-sst-2-english

This model is a fine-tune checkpoint of [DistilBERT-base-uncased](https://huggingface.co/distilbert-base-uncased), fine-tuned on SST-2. This model reaches an accuracy of 91.3 on the dev set (for comparison, Bert bert-base-uncased version reaches an accuracy of 92.7).

This model can be used for topic classification. You can use the raw model for either masked language modeling or next sentence prediction, but it's mostly intended to be fine-tuned on a downstream task.

HuggingFace Link: https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english
